### PR TITLE
Fixing two issues encountered during the setup of development environment in Ubuntu OS

### PR DESCRIPTION
**Pull request by shivamriky**
_modified:   modules/ide-integration/src/main/resources/airavata-server.properties_
Updated changed the server.host from localhost to 0.0.0.0 as the localhost as localhost was creating problems in Linux where
the docker container was not able to talk to the API services due to a different host address.

_modified:   pom.xml_
1. Added maven repository as maven was not able to download some libraries from already available repos.
2. Added a plugin for openJPA to avoid the following error in Intellij:
"<openjpa-2.4.3 nonfatal user error> org.apache.openjpa.persistence.ArgumentException: This configuration disallows runtime optimization, but the following listed types were not enhanced at build time or at class load time with a javaagent: "

### CODE REVIEW

**Comment by shivamriky** [comment]
Just for reference, the following file contains the error I was getting before adding the new dependency. It also includes the successful build message after adding the new dependency. 
[maven_build_error.txt](https://github.com/apache/airavata/files/3204071/maven_build_error.txt)

**Comment by DImuthuUpe** [comment]
Thanks @shivamriky. I will merge this

File: pom.xml [hunk 1]
```diff
@@ -552,6 +552,11 @@
                 <enabled>false</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>wso2</id>
+            <name>Repository</name>
+            <url>http://dist.wso2.org/maven2</url>
```
Comment (machristie) [inline pom.xml]: Hi @shivamriky,

Can you change this url to use `https` instead?

Comment (machristie) [reply pom.xml]: I went ahead and made this change.

Comment (shivamriky) [reply pom.xml]: Thank you. I didn't see the message earlier.

**Pull request merged**